doc/samtools-mpileup.1

```diff
@@ -257,6 +257,12 @@ Disable read-pair overlap detection.
 Include customized index file as a part of arugments. See
 .B EXAMPLES
 section for sample of useage.
+.TP
+.B --reverse-del
+Mark the deletions on the reverse strand with the character
+.B #
```

Comment:
Use `.BR # ,` to avoid the space between `#` and `,`, and similarly for the fullstop after `the usual *`.

This should probably be listed under _Output Options_ (and similarly in the Usage string).